src/Runner/TestSuiteSorter.php

```diff
@@ -232,15 +205,17 @@ private function addSuiteToDefectSortOrder(TestSuite $suite): void
         $max = 0;
 
         foreach ($suite->tests() as $test) {
-            $testname = self::getTestSorterUID($test);
+            if (!($test instanceof Reorderable)) {
```

Comment:
```suggestion
            if (!$test instanceof Reorderable) {
```

Reply:
Weird cs fixer didn't pick this up. Can we activate a rule for this perhaps?